not-supported/tpm2-unlock.sh

```diff
@@ -44,7 +44,7 @@ fi
 
 # Test if device is present
 # https://github.com/noobient/noobuntu/wiki/Full-Disk-Encryption#tpm-2
-if ! dmesg | grep -i "tpm" | grep -q "2\.0"
+if ! dmesg | grep -i "tpm" | grep -q "2"
```

Comment:
Maybe?
```suggestion
if ! dmesg | grep -i "tpm2"
```

Reply:
I don't know, needs to be tested.

Reply:
Yes, will try to do so when I am back home. Currently traveling

Reply:
@J-Kappes can you post the output of `dmesg | grep -i "tpm"` here?
On my server at home this is the output:
```
[    1.139024] ima: No TPM chip found, activating TPM-bypass!
[    2.399415] systemd[1]: systemd 255.4-1ubuntu8.4 running in system mode (+PAM +AUDIT +SELINUX +APPARMOR +IMA +SMACK +SECCOMP +GCRYPT -GNUTLS +OPENSSL +ACL +BLKID +CURL +ELFUTILS +FIDO2 +IDN2 -IDN +IPTC +KMOD +LIBCRYPTSETUP +LIBFDISK +PCRE2 -PWQUALITY +P11KIT +QRENCODE +TPM2 +BZIP2 +LZ4 +XZ +ZLIB +ZSTD -BPF_FRAMEWORK -XKBCOMMON +UTMP +SYSVINIT default-hierarchy=unified)
[    3.004319] systemd[1]: systemd-pcrextend.socket - TPM2 PCR Extension (Varlink) was skipped because of an unmet condition check (ConditionSecurity=measured-uki).
[    3.188863] systemd[1]: systemd-pcrmachine.service - TPM2 PCR Machine ID Measurement was skipped because of an unmet condition check (ConditionSecurity=measured-uki).
[    3.199485] systemd[1]: systemd-tpm2-setup-early.service - TPM2 SRK Setup (Early) was skipped because of an unmet condition check (ConditionSecurity=measured-uki).
```
It is weird because I am sure that my server supports tpm2... Probably something in the kernel or a package broke...

Reply:
```
[    0.000000] efi: TPMFinalLog=0x78d84000 ACPI=0x78e2b000 ACPI 2.0=0x78e2b014 SMBIOS=0x75e53000 SMBIOS 3.0=0x75e51000 ESRT=0x75e3e398 MEMATTR=0x72ef7018 MOKvar=0x75e36000 RNG=0x78df5018 TPMEventLog=0x66818018 
[    0.016671] ACPI: TPM2 0x0000000078E26000 000034 (v04 HPQOEM 84D4     00000002 HP   00040000)
[    0.017028] ACPI: Reserving TPM2 table memory at [mem 0x78e26000-0x78e26033]
```

Reply:
That seems like a safer solution if you ask me.

@szaimen merge?